rbd/snapshot_nautilus.go

```diff
@@ -65,6 +65,64 @@ func (image *Image) GetParentInfo(pool, name, snapname []byte) error {
 	return nil
 }
 
+// ImageSpec represents the image information.
+type ImageSpec struct {
+	imageName string
```

Comment:
The members of these types are private to this package. Is that intentional? I do not see how callers of `GetParent()` can use these types.

Reply:
Sorry, its by mistake. Will fix it.